cdap-ui/app/cdap/components/Overview/OverviewHeader/index.js

```diff
@@ -41,7 +41,7 @@ export default class OverviewHeader extends Component {
   }
   render() {
     return (
-      <div className="overview-header">
+      <div className={classnames("overview-header", {'message-placeholder': typeof this.props.onClose !== 'function'})}>
```

Comment:
So in entity detailed view we don't need it. When we don't have any content we still add padding to the overview-header which occupies un-necessary space. This is to avoid it. If there is no `onClose` prop then we are not showing anything in overview-header. The `message-placeholder` removes padding.